lib/tasks/taskRepository.js

```diff
@@ -42,6 +75,55 @@ export async function getTask(taskName) {
 	}
 }
 
+/**
+ * Returns a list of the names of all available tasks
+ *
+ * @private
+ * @static
+ * @returns {string[]} Array containing the names of all available tasks
+ */
 export function getAllTaskNames() {
 	return Object.keys(taskInfos);
 }
+
+/**
+ * Returns a list of the names of all tasks that have been removed
+ * in this or previous versions of ui5-builder.
+ *
+ * @private
+ * @static
+ * @returns {string[]} Array containing the names of all removed tasks
+ */
+export function getRemovedTaskNames() {
+	return ["createDebugFiles", "uglify", "generateManifestBundle"];
```

Comment:
in future we might change the array of strings to an array of objects providing also the information with which version the task got removed. For now, its fine. But for future `v4` it might be relevant.